config/settings/development.yml

```diff
@@ -1,6 +1,7 @@
 SECRET_KEY_BASE: 6c51f19b57fdebd3dfd2a3cf8b69fab7f41a529b6bfbf8bff4a85815934066cbe22ddf7fa6083f227288c04e4eeb6c28f0fd90e31fc02314aaceb135dacecc07
 
 DEVISE_SECRET_TOKEN: 2c847492b6d434c808e63a93ff16a7a897aa4c09943edaaf7aa498a29649048113cf55731282e39a82ae7ce3fb9f1087586d474fc483ad6f63806986a5a02a6c
+NEWRELIC_LICENSE_KEY: none
```

Comment:
To avoid putting the license key in Git, I'm setting this to none and disabling monitoring in development.